### DISCUSSION TITLE
Why bots love to sit in chairs but never lie down on the bed?

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13582] [Ideas and suggestions]
Why bots can't use beds?
It doesn't matter on a boat or at a station he is.
A wounded bot, if he does not have active orders, must look for a free bed and lie down on it. This will save us medicines.
This will be useful in both single-player and network game.

**Comment by BlackMoons** [comment]
And can we tell bots to stop sitting in certain chairs? They keep stealing my captains chair...

**Reply by pavel4win** [reply]
It's very easy, just click on the captain's  chair with the middle mouse button and give the command "ignore"

**Comment by pavel4win** [comment]
Please make it possible for an unconscious player or bot to be put on a bed by other player.
If an unconscious player can be carried, then he can be placed on a bed or placed in a chair, this is logical.
